lib/common/actions.c

```diff
@@ -144,37 +144,37 @@ pcmk__parse_action(const char *action_name)
  * \return Static string describing \p on_fail
  */
 const char *
-pcmk__on_fail_text(enum action_fail_response on_fail)
+pcmk__on_fail_text(enum pcmk__on_fail on_fail)
```

Comment:
Not new but this function should also have `\internal` in doxygen

Reply:
thx

Reply:
ah I commented on this twice. GitHub didn't show the earlier comment when I made this one, so I assumed this was an old function the second time.